ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/reader/interfaces/IValidationStrategy.java

```diff
@@ -0,0 +1,7 @@
+package edu.usb.argos.ASTProcessor.reader.interfaces;
+
+import edu.usb.argos.ASTProcessor.reader.exceptions.FileAnalyzerException;
+
+public interface IValidationStrategy<T> {
```

Comment:
The name is generic, maybe it could be ISourceValidationStrategy or similar because it is also referring to exceptions related to files.

Reply:
Done